src/main/java/eu/darkbot/api/PluginAPI.java

```diff
@@ -23,6 +30,17 @@
      */
     @NotNull <T extends API> T requireAPI(@NotNull Class<T> api) throws UnsupportedOperationException;
 
+    /**
+     * @return current used {@link Module}
+     */
+    Module getModule();
```

Comment:
the plugin api doesn't hold the module, the "main" or the "bot" does, consider it an api that you can get from PluginAPI, like `pluginApi.requireAPI(DarkBot.class).getModule()` ?